config/initializers/geocoder.rb

```diff
@@ -0,0 +1,30 @@
+Geocoder.configure(
+  # per Geocoder documentation, specifying a header while using default service Nominatim
+  # https://github.com/alexreisner/geocoder/blob/master/README_API_GUIDE.md#nominatim-nominatim
+  http_headers: { "User-Agent" => "steven@looseends.org" }
```

Comment:
Is this right?

Reply:
Great catch, I need to change this to looseendsproject, or ideally, some group email we'd want to send alerts of this nature to.